File: lua/primes.lua
local n = 100000 -- incorrect
local primes = {2}

for i = 3, n do
	local isPrime = true
	for j = 1, #primes do
		if i % primes[j] == 0 then
			isPrime = false
			break
		end
	end
	if isPrime then
		table.insert(primes, i)
	end
end

File: python2/primes.py
primes = [2]
for i in xrange(3, 100000): # Incorrect range!
    is_prime = True
    for j in xrange(0, len(primes)):
        p = primes[j]
        if j*j > i:
            break
        if p % i == 0:
            is_prime = False
            break
    if is_prime:
        primes.append(i)
